src/ui/manager.tsx

```diff
@@ -26,21 +21,9 @@ const ManagerUI: React.FC<ReaderReturn> = ({
           flexDirection: 'column',
         }}
       >
-        <nav
-          style={{
-            display: 'flex',
-            justifyContent: 'space-between',
-            alignItems: 'center',
-            padding: 8,
-          }}
-        >
-          <h1>{manifest?.metadata.title}</h1>
-          <div>
-            {navigator && state && (
-              <SettingsCard navigator={navigator} state={state} />
-            )}
-          </div>
-        </nav>
+        {navigator && state && (
```

Comment:
What does it mean when either navigator or state are not present here?  do we need to throw an error?

Reply:
It means the App is still loading the content, [initially null](https://github.com/NYPL-Simplified/web-reader/blob/48f0854796577e8514a6b0639504d273c8a0e1ce/src/useWebReader.tsx#L83)